### PR TITLE
feat(volo): scrimba added

### PR DESCRIPTION
**Pull request by Martyniuk**

### CODE REVIEW

File: volo/scrimba/passwordGenerator/scripts.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+  let optionOneValue = "";
+  let optionTwoValue = "";
+  const passwordLength = Number(passwordLengthInput.value) || 15;
+  const includeSymbols = symbolsToggler.checked;
+  const includeNumbers = numbersToggler.checked;
+
+  if (includeSymbols) {
+    characters.push(...charactersObj.symbols);
+  }
+  // } else {
```
Comment (ArtemVMV) [inline volo/scrimba/passwordGenerator/scripts.js:120]: WTF?!?!?!?!

**Review by ArtemVMV — CHANGES_REQUESTED** [review]
There is a commented code line

**Review by ArtemVMV — APPROVED** [review]
👍🏿

File: volo/scrimba/passwordGenerator/scripts.js (hunk 1 above)
Comment (Martyniuk) [reply volo/scrimba/passwordGenerator/scripts.js]: good spot, fixed

**Pull request merged**